src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/mockservers/HealthServer.kt

```diff
@@ -21,11 +25,39 @@ open class HealthServer(config: WireMockConfiguration, private val healthCheckPa
 
```

Comment:
Just noticed: I'd prefer this class to be `HealthMockServer` - or somehow to be named as a mock.